src/lib/mdns/Resolver.h

```diff
@@ -107,14 +111,17 @@ enum class DiscoveryFilterType : uint8_t
     kDeviceType,
     kCommissioningMode,
     kCommissioningModeFromCommand,
+    kInstanceName,
     kCommissioner
 };
 struct DiscoveryFilter
 {
     DiscoveryFilterType type;
     uint16_t code;
+    char * instanceName;
     DiscoveryFilter() : type(DiscoveryFilterType::kNone), code(0) {}
     DiscoveryFilter(DiscoveryFilterType newType, uint16_t newCode) : type(newType), code(newCode) {}
+    DiscoveryFilter(DiscoveryFilterType newType, char * newInstanceName) : type(newType), instanceName(newInstanceName) {}
```

Comment:
Oh, one more question.  Should `newInstanceName` (and the `instanceName` member) be `const char *` so that string literals can be passed in?